tests/testDipole.py

```diff
@@ -80,10 +80,10 @@ def createDipole(w, h, xc, yc, scaling=100.0, fracOffset=1.2):
     offset = fracOffset * psfFwhmPix // 2
     array = image.getImage().getArray()
     xp, yp = xc - psfw//2 + offset, yc - psfh//2 + offset
-    array[yp:yp+psfh, xp:xp+psfw] += psfim.getArray()
+    array[int(yp):int(yp)+psfh, int(xp):int(xp)+psfw] += psfim.getArray()
```

Comment:
in some sense
```python
xp = int(xc - psfw//2 + offset)
yp = int(yc - psfh//2 + offset)
```
is more readable because you can easily spot the differences between `xp` and `yp` by scanning down the screen.